global_setup/groups.tf

```diff
@@ -0,0 +1,12 @@
+locals {
```

Comment:
@samkahchiin  can we name the folder `global-setup` with a dash? For other modules, we used a dash so far.